doc/howto/storage_volumes.md

```diff
@@ -131,6 +156,40 @@ All I/O limits only apply to actual block device access.
 Therefore, consider the file system's own overhead when setting limits.
 Access to cached data is not affected by the limit.
 
+````
+```` {group-tab} UI
+To attach a storage volume to an instance, navigate to the configuration page of the target instance.
+
+In the secondary menu, select {guilabel}`Disk` under {guilabel}`Devices`.
```

Comment:
The Order of the gui labels should follow the path the user takes, first devices, then disk.